src/Engine/ProtoAssociative/CodeGen.cs

```diff
@@ -2195,6 +2240,7 @@ private void SSAIdentList(AssociativeNode node, ref Stack<AssociativeNode> ssaSt
                     bnode.Optr = ProtoCore.DSASM.Operator.assign;
                     bnode.isSSAAssignment = true;
                     bnode.isSSAPointerAssignment = true;
+                    bnode.IsFirstIdentListNode = true;
```

Comment:
Sure, the AST replacer is certainly a cleaner approach. 
For this I will have to explore using the AstReplacer for the SSA transform.
The SSAIdentList is only one small part of BuildSSA(). I want to make sure that the entire SSA transform algorithm is refactored to use the applicable patterns.

Pls see: http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-7435